packages/orbit-components/src/StopoverArrow/index.js

```diff
@@ -46,7 +46,14 @@ const Stops = ({ stops }: Props) => {
 };
 
 const StopoverArrow = ({ stops = STOPS.ZERO, dataTest }: Props) => (
-  <StyledArrow preserveAspectRatio="xMidYMid meet" viewBox="0 0 28 8" data-test={dataTest}>
+  <StyledArrow
+    preserveAspectRatio="xMidYMid meet"
+    viewBox="0 0 28 8"
+    data-test={dataTest}
+    role="img"
```

Comment:
Addition of `role` should be a `fix` commit, and you can add an assertion for it in the test. I'd also check `<title>` and `<desc>` and connected them with `aria-labelledby` like you did before.